src/main/java/org/sunbird/cassandra/utils/CassandraOperationImpl.java

```diff
@@ -225,8 +227,10 @@ public Map<String, Object> updateRecord(String keyspaceName, String tableName, M
 			session.execute(updateQuery);
 			response.put(Constants.RESPONSE, Constants.SUCCESS);
 		} catch (Exception e) {
-			logger.error(e.getMessage());
-			throw e;
```

Comment:
Adding more details in response object is fine... but do not change the behaviour without any analysis... you have removed "throw e" -- that is going affect existing implementation